airflow/www/views.py

```diff
@@ -3331,7 +3331,7 @@ def grid_data(self):
             base_date = dag.get_latest_logical_date() or timezone.utcnow()
 
         with create_session() as session:
-            query = select(DagRun).where(DagRun.dag_id == dag.dag_id, DagRun.logical_date <= base_date)
+            query = select(DagRun).where(DagRun.dag_id == dag.dag_id, DagRun.run_after <= base_date)
```

Comment:
Probably don’t change this. Not sure if we want to keep base_date (I threw a question to Brent and Pierre about this), but this needs to be re-implemented in new UI anyway, and we can figure things out there.